ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Node/Serialisation.hs

```diff
@@ -65,7 +65,7 @@ instance EncodeDisk ByronBlock (PBftState PBftByronCrypto) where
   encodeDisk _ = encodeByronChainDepState
 -- | @'ChainDepState' ('BlockProtocol' 'ByronBlock')@
 instance DecodeDisk ByronBlock (PBftState PBftByronCrypto) where
-  decodeDisk ccfg = decodeByronChainDepState (getByronSecurityParam ccfg)
+  decodeDisk _ = decodeByronChainDepState
```

Comment:
Extend the golden test infrastructure to be able to test backwards compatibility of things. Use in combination with `Versioned`? Create an issue.

Reply:
See #2596.